azure-bootstrap.yml

```
@@ -0,0 +1,169 @@
+- job: Trigger
+  displayName: Trigger Azure Dev Ops build and test pipeline
+  pool:
+    vmImage: 'ubuntu-latest'
+
+  variables:
+    AZURE_DEVOPS_ORG: nanoFramework
+    AZURE_DEVOPS_PROJECT: nanoFramework.Protobuf
+    AZURE_DEVOPS_PIPELINE_ID: 117
+    AZURE_POOL_NAME: TestStream    
+
+  steps:   
```

yamllint

[error] trailing spaces

(trailing-spaces)